.travis.yml

```diff
@@ -9,12 +11,25 @@ sudo: false
 # back to only build the stage1 compiler and run a subset of tests, but this
 # didn't end up panning out very well.
 #
-# As a result, we're just using travis to run `make tidy` now. It'll help
-# everyone find out about their trailing spaces early on!
+# As a result, we're just using travis to run `make tidy` and *only* build
+# stage1 but *not* test it for now (a strict subset of the bootstrap). This will
+# catch "obvious" errors like style or not even compiling.
+
 before_script:
-  - ./configure --llvm-root=path/to/nowhere
+  - ./configure --enable-ccache
 script:
   - make tidy
+  - make rustc-stage1 -j8
```

Comment:
I'd expect the normal context switching cost. Maybe worth testing?
On Jul 14, 2015 11:19 AM, "Alexis Beingessner" <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/rust-lang/rust/pull/27028#discussion_r34579851>:
>
> >  script:
> >    - make tidy
> > +  - make rustc-stage1 -j8
>
> Is there a negative impact of giving too big of a number? In my experience
> you just don't gain anything.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rust-lang/rust/pull/27028/files#r34579851>.
>